basics/indexing/dictionary-index.md

```diff
@@ -0,0 +1,124 @@
+        }
+      }
+    },
+    ...
+  ],
+...
+}
+```
+{% endcode %}
+
+Alternatively, the `encodingType` property can be changed. For example:
+```javascript
+{
+  "fieldConfigList": [
+    {
+      "name": "theTableName",
+      "encodingType": "RAW"
+    },
+    ...
+  ],
+...
+}
+```
+{% endcode %}
+
+Use the option you prefer, but try to be consistent, as Pinot will reject table configurations where the same column and
+index is defined in different places.
+
+The decision of actually create the dictionary or not can be delegated to some included Pinot heuristics.
+This feature can be enabled in `indexingConfig` object inside table config.
+The parameters that control the heuristic are:
+
+
+| Parameter                      | Default | Description                                                          |
+|--------------------------------|---------|----------------------------------------------------------------------|
+| optimizeDictionary             | false   | Enables the heuristic for all columns and activate some extra rules. |
+| optimizeDictionaryForMetrics   | false   | Enables the heuristic for metric columns.                            |
+| noDictionarySizeRatioThreshold | 0.85    | The ratio used in the heuristics.                                    |
+
+It is important to note that these parameters are configured for all columns on the table.
+When this optimization is enabled, columns explicitly marked as dictionary will actually be encoded as raw 
+if:
+- They are not multi-valued.
+- And their column type is fixed size (like int, long, double, timestamp, etc).
+- And their ratio between the forward index size encoded as raw and the forward index size encoded as dictionary is lower than `noDictionarySizeRatioThreshold`.
+
+If `optimizeDictionary` is false and `optimizeDictionaryForMetrics` is true, only columns declared as metric will be affected.
+If `optimizeDictionary` is true, all columns will be affected.
+Also in this case columns indexed as [text](./text-search-support.md) or [JSON](./json-index.md) will automatically be indexed as raw.
+Note that `optimizeDictionary` has priority over `optimizeDictionaryForMetrics` and the special rule applied to text 
+and JSON indexes only applies when `optimizeDictionary` is true.
+
+### Parameters
+
+There are some options you can configure:
+
+| Parameter              | Default | Description                                    |
+|------------------------|---------|------------------------------------------------|
+| onHeap                 | false   | If the index must be load on heap or off heap. |
+| useVarLengthDictionary | false   | How to store variable length values.           |
+
+Dictionaries are always off heap, but in cases where the cardinality is small and the footprint on heap is acceptably 
+small, they can be copied in memory by changing `onHeap` parameter to true.
+When they are on heap, dictionaries can be faster and some extra optimizations can be done.
```

Comment:
helpful to clarify "they"

Reply:
I don't get it. We are talking about dictionaries here. Is the pronoun not correctly used here?